hangman/lib/database.py

```diff
@@ -7,9 +7,9 @@ def create_table():
     """sqlite table created to store guesses"""
     db.execute("CREATE TABLE IF NOT EXISTS guesses(letter CHAR)")
 
-def guesses_table(stored_guessed_letter):
+def guesses_table(store_guessed_letter):
```

Comment:
No. I mean rename the function. The variable  name was ok as `guess`.

Reply:
@yudori done